### DISCUSSION TITLE
Git/GitHub Integration for Document Revision History

### DISCUSSION THREAD

**Discussion by ahsnuet** [texstudio-org/texstudio#3884] [Ideas]
Proposing the Git/GitHub integration in TeXstudio to enable tracking document revision history directly within the editor. This feature would provide users with version control capabilities akin to Overleaf's paid version, allowing them to commit, push, pull, and view diffs without leaving TeXstudio.

**Comment by sunderme** [comment]
you have seen this: https://texstudio-org.github.io/advanced.html#git-svn-support ?

**Reply by ahsnuet** [reply]
Thanks for sharing, perhaps I skipped this part during search.
